larray/core/ufuncs.py

```diff
@@ -302,3 +303,36 @@ def broadcastify(func):
      a0   6   2   5   0
      a1   5   2   3   0
 """
+
+
+def isscalar(element: Any) -> bool:
```

Comment:
is isscalar an ufunc (numpy universal function)? I don't think it is, so moving this to another module would be a good idea I think.

Reply:
Correct. I move it to util/misc.py or do I create a new module in the `core` directory?

Reply:
AFAIR, util/misc.py are for internal functions not api-exposed functions, so I wouldn't put it there. Maybe core/misc.py?

Reply:
OK for core/misc.py